data-service/src/main/java/com/amazonaws/blox/dataservice/model/Environment.java

```diff
@@ -38,7 +38,7 @@
   @NonNull private EnvironmentStatus environmentStatus;
   @NonNull private String latestEnvironmentRevisionId;
   // environment and revision were both successfully created
-  @NonNull private boolean validEnvironment;
+  private boolean validEnvironment;
```

Comment:
Why do we remove the @NonNull for this variable

Reply:
It's a primitive type, so never be null and it cause a build warning.